asmcomp/spacetime_profiling.ml

```diff
@@ -34,7 +34,7 @@ let reverse_shape = ref ([] : Mach.spacetime_shape)
    in [Cmmgen]. *)
 let cconst_int i = Cmm.Cconst_int (i, Debuginfo.none)
 let cconst_natint i = Cmm.Cconst_natint (i, Debuginfo.none)
-let cconst_symbol s = Cmm.Cconst_symbol (s, Debuginfo.none)
+let cconst_symbol s = Cmm.Cconst_symbol (s, Cmm.typ_val, Debuginfo.none)
```

Comment:
I think this should be `typ_int` -- I would also change `cconst_symbol` to `cconst_int_symbol` or something to make this clear.

Reply:
done